open-source-workflow.html

```diff
@@ -99,7 +99,7 @@ <h4 class="requirements">Testing</h4>
 
   <div class="workflowBlush5 workflowChart">
         <h4 class="requirements">Maintenance</h4>
-        <p>Create Issues that Fix Errors/Improve Impelentation/Upgrade the Current Language <i class="fas fa-long-arrow-alt-right"></i> Approved by Mentor/Maintainer/Committer</p>
+        <p>Create Issues that Fix Errors/Improve Implementation/Upgrade the Current Language <i class="fas fa-long-arrow-alt-right"></i> Approved by Mentor/Maintainer/Committer</p>
```

Comment:
@AnkitKhatri55555 because the https://github.com/systers/systers.github.io/pull/73 was created from Master, and you have rebased form your master, you are seeing this change. You need to fix your master first. and then revert changes in this file. And then finally squash commits.